client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -431,41 +446,42 @@ function mediaButton( editor ) {
 		editor.nodeChanged();
 	}
 
-	function toggleSizingControls( event ) {
+	function toggleSizingControls( increase, event ) {
 		if ( ! event.element || 'IMG' !== event.element.nodeName ) {
 			return;
 		}
 
+		const site = sites.getSelectedSite();
 		const parsed = MediaSerialization.deserialize( event.element );
+		const media = assign( { width: Infinity, height: Infinity }, MediaStore.get( site.ID, parsed.media.ID ) );
 
-		// Disable sizing toggles if the image is transient
-		let isHidden = parsed.media.transient;
+		// Hide sizing toggles if the image is transient
+		const isHidden = parsed.media.transient || false;
```

Comment:
How about `!! parsed.media.transient` instead of the fallback? While we know in this case that `transient` will only ever be `true`, `false`, or `undefined`, being explicit about converting it to a boolean helps communicate your intent. And if `transient` were to ever change shape, this also ensures that the value is always a boolean, because currently truthy values would be assigned as their original value.

There's also `Boolean( parsed.media.transient )` if the double-"not" feels too magical to you.